.github/workflows/cleanup-test-env.yml

```diff
@@ -1,9 +1,8 @@
 name: 'Cleanup test env'
 
 on:
-  workflow_dispatch: {} # workflow can be run manually
-  schedule:
-    - cron: "0 3 * * *" # workflow runs every day at 03:00 AM
```

Comment:
don't schedule clean-up job as it's called from Test Suite so they don't interfere